src/components/TranslateMarkdown.tsx

```diff
@@ -11,15 +11,15 @@ interface Props {
 export const TranslateMarkdown = ({ source }: Props) => {
   return (
     <Markdown
-      escapeHtml={true}
-      unwrapDisallowed={true}
-      allowedTypes={['link', 'emphasis', 'strong', 'code', 'root', 'inlineCode']}
+      disallowedTypes={['html']}
```

Comment:
Do we not want to stick to the currently allowed types?

Reply:
For un unknown reason none of our translation are rendered when using `allowedTypes` so we use the inverse just to ban what is potentially dangerous.